CHANGELOG.md

```diff
@@ -1,5 +1,66 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.2.2
+
+<!-- generated comparing v5.2.1..master -->
+
+_Nov 29, 2021_
+
+A big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- ♿️ Improved accessibility of `Snackbar` and `TextField` in `@mui/material` (#29782) (#29850) (#29852) @eps1lon.
+- 🎉 Finished the first iteration of `Button` of `Joy` design system (#29464) @siriwatknp.
```

Comment:
I think it's not time to announce anything Joy-related already on the release notes, I'd keep it a little under the radar as of now, at least before the first marketing page is out! What do you think?

Reply:
Yep, it makes sense! Will remove it!